src/tailwind.css

```diff
@@ -3,6 +3,37 @@
 @tailwind components;
 /* purgecss end ignore */
 
+.btn {
+  @apply font-bold py-1 px-4 rounded;
+}
+
+.btn-blue {
+  @apply bg-blue-500 text-white;
+}
+.btn-blue:hover {
+  @apply bg-blue-700;
+}
+
+.btn-darkBlue {
+  @apply bg-blue-800 text-white;
+}
+.btn-darkBlue:hover {
+  @apply bg-blue-700;
+}
+
+.btn-TransparentBlue {
+  @apply bg-transparent text-black border-solid border-2 border-gray-500;
+}
+.btn-TransparentBlue:hover {
+  @apply bg-gray-200 border-solid border-2 border-gray-500;
+}
+
+.btn-redXBtn {
```

Comment:
this is also a one off component, and would make more sense if it was applied inside the RedXBtn.js directly